src/Fable.Transforms/Replacements.fs

```diff
@@ -2904,7 +2904,8 @@ let guids (com: ICompiler) (ctx: Context) (r: SourceLocation option) t (i: CallI
 
 let uris (com: ICompiler) (ctx: Context) (r: SourceLocation option) t (i: CallInfo) (thisArg: Expr option) (args: Expr list) =
     match i.CompiledName with
-    | ".ctor" -> Helper.LibCall(com, "Uri", "default", t, args, i.SignatureArgTypes, isJsConstructor=true, ?loc=r) |> Some
+    | ".ctor" -> Helper.LibCall(com, "Uri", "Uri.create", t, args, i.SignatureArgTypes, ?loc=r) |> Some
```

Comment:
I'm particularly interested in feedback on this bit as it's the most foreign to me. Specifically, is it OK to call the static `Uri.create` method in JS even though this is a constructor in .NET. The reason I made it a static create method in the JavaScript implementation was because it allowed me to better share the creation logic with `TryCreate` and also allowed me to make better use of TypeScript's type system to eliminate some invalid parameter combinations.

Reply:
The Replacements module and fable-library are still a bit messy. It's great that you managed to find your way through it! :clap: Ideally we want to progressively update it so most of the code is in F#, but given the Uri module is still in Typescript and create/tryCreate is similar to parse/tryParse in other modules, your implementation looks good to me 👍

Reply:
Thanks. I'll crack on and write the tests too then.